### PR TITLE
Change admin login credentials

### PR DESCRIPTION
**Pull request by jamestjw**

### CODE REVIEW

File: install.php [hunk 1]
```diff
@@ -31,10 +31,10 @@
 $admin = new User();
 $admin->first_name = "admin";
 $admin->last_name = "user";
-$admin->email = "admin@concordia.ca";
+$admin->email = "admin";
 $admin->is_admin = 1;
 $admin->is_instructor = 0;
-$admin->password_digest = password_hash('root', PASSWORD_DEFAULT);
```
Comment (drouba) [inline install.php:37]: I think the password he requested was admin no?

**Review by drouba — CHANGES_REQUESTED** [review]

File: install.php (hunk 1 above)
Comment (drouba) [reply install.php:37]: yup checked its admin admin

Comment (jamestjw) [reply install.php:37]: @drouba so all good right?

**Review by drouba — APPROVED** [review]
all good

**Pull request merged**